packages/loader/container-utils/src/liveSummarySanitizer.ts

```diff
@@ -0,0 +1,194 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    ICommittedProposal,
+    ISummaryTree,
+    SummaryType,
+    ISummaryBlob,
+} from "@fluidframework/protocol-definitions";
+
+import { assert, bufferToString } from "@fluidframework/common-utils";
+
+function readBlobContent(content: ISummaryBlob["content"]): any {
+    const json =
+        typeof content === "string" ? content : bufferToString(content, "utf8");
+    return JSON.parse(json);
+}
+
+function validateKeyPresence(
```

Comment:
Can you please add comments on top of each function?

Reply:
We are moving away from `liveSummarySanitizer`.